learn/what_is_meilisearch/sdks.mdx

```diff
@@ -30,7 +30,8 @@ You can use Meilisearch API wrappers in your favorite language. These libraries
 - [Angular](https://github.com/meilisearch/meilisearch-angular)
 - [React](https://github.com/meilisearch/meilisearch-react)
 - [Vue](https://github.com/meilisearch/meilisearch-vue)
-- [Instant Meilisearch](https://github.com/meilisearch/instant-meilisearch): helps you integrate a great search experience with minimum efforts.
+- [Instant Meilisearch](https://github.com/meilisearch/meilisearch-js-plugins/tree/main/packages/instant-meilisearch): helps you integrate a great search experience with [`instantsearch.js`](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/js/).
+- [Autocomplete Client](https://github.com/meilisearch/meilisearch-js-plugins/tree/main/packages/autocomplete-client): helps you integrate a great search experience with [`autocomplete-js`](https://www.algolia.com/doc/ui-libraries/autocomplete/introduction/what-is-autocomplete/).
```

Comment:
We don't describe the other tools, so I think we can skip these too. 

From what I understand these are targeted at users who are already familiar with Instant Search and Autocomplete, so we should be fine?

```suggestion
- [Instant Meilisearch](https://github.com/meilisearch/meilisearch-js-plugins/tree/main/packages/instant-meilisearch)
- [Autocomplete client](https://github.com/meilisearch/meilisearch-js-plugins/tree/main/packages/autocomplete-client)
```